README.md

```diff
@@ -1,124 +1,115 @@
 
-```bash
-# Clone github repository.
-$ git clone https://github.com/milvus-io/milvus.git
 
-# Install third-party dependencies.
-$ cd milvus/
-$ ./scripts/install_deps.sh
+**Support for Various Vector Index Types and Hardware Acceleration**  
+  * Milvus supports vector index types that are optimized for different scenarios, including HNSW, IVF, FLAT (brute-force), and DiskANN, with [quantization-based](https://milvus.io/docs/index.md?tab=floating#IVFPQ) variations and [mmap](https://milvus.io/docs/mmap.md). Additionally, Milvus implements instruction-level acceleration to enhance vector search performance and supports GPU indexing, such as NVIDIA's [CAGRA](https://github.com/rapidsai/raft).
 
-# Compile Milvus.
-$ make
-```
+**Sparse Vector for Full Text Search and Hybrid Search**
+  * In addition to semantic search via dense vector embeddings, Milvus also supports full text search with [sparse vector](https://milvus.io/docs/sparse_vector.md). User can also combine dense vector and sparse vector through multi-vector feature and perform hybrid search. Milvus supports up to 10 vector fields in a single collection. Milvus supports hybrid search on dense and sparse vector columns, or multiple dense vector columns. The data from multi-path retrieval can be merge and rerank with Reciprocal Rank Fusion (RRF) and Weighted Scoring. For details, refer to [Hybrid Search](https://milvus.io/docs/multi-vector-search.md).
+
+**Flexible Strategies for Multi-tenancy**
+  * Milvus supports [multi-tenancy](https://milvus.io/docs/multi_tenancy.md#Multi-tenancy-strategies) with flexible strategies to provide a robust solution for organizing data in AI applications, particularly in Retrieval-Augmented Generation (RAG) systems for both enterprise use cases and consumer apps. By leveraging database, collection, and partition key structures, Milvus allows businesses to allocate specific resources to each tenant, enabling flexible data isolation, optimized search performance, and scalable access management. This approach suits large-scale knowledge bases for enterprises, with strong isolation at the database level, and high user capacity for consumer applications by partitioning data within shared collections.
```

Comment:
This approach suits large-scale knowledge bases for enterprises, with strong isolation at the database level, and high user capacity for consumer applications by partitioning data within shared collections.

Don't quite understand the meaning of this?
Maybe we can remove this and add hot cold data separation for different tenatns